lib/smack/SmackInstGenerator.cpp

```diff
@@ -490,15 +490,20 @@ void SmackInstGenerator::visitSelectInst(llvm::SelectInst& i) {
   processInstruction(i);
   std::string x = naming->get(i);
   const Expr
-  *c = rep->expr(i.getOperand(0)),
-   *v1 = rep->expr(i.getOperand(1)),
-    *v2 = rep->expr(i.getOperand(2));
+  *c = rep->expr(i.getCondition()),
+   *v1 = rep->expr(i.getTrueValue()),
+    *v2 = rep->expr(i.getFalseValue());
 
+  /*
```

Comment:
Why don't you completely erase this instead of just commenting it out?

Reply:
It's not yet known if the new encoding would influence performance. So I keep the old encoding for reference.

Reply:
Ok, got it, well, we should not merge until we figure it out and you erase it.